### PR TITLE
docs: add litestar, fix some grammar and spacing

### PR DESCRIPTION
**Pull request by JacobCoffee**
**Problem**
Currently, `Litestar` is not listed

**Explanation**
This happens because someone forgot to add it

**Solution**
With this PR we add Litestar, as well as adjust some grammatical issues and add GitHub's new admonition syntax for notes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,29 +18,31 @@ Generate interactive API documentations from Swagger files. [Try our Demo](https
 - Edit your Swagger files with a live preview
 - Doesn’t look like it’s 2011
```
Comment (JacobCoffee) [inline README.md:20]: New admonition syntax: https://github.com/orgs/community/discussions/16925

File: README.md [hunk 2]
```diff
@@ -111,14 +115,14 @@ If you’d like to add a request proxy for the API client (to avoid CORS issues)
...
-You can easily run Scalar API References in Nuxt via the module.
+You can easily run Scalar API References in Nuxt via the module:
 
 ```bash
 npx nuxi module add @scalar/nuxt
 ```
 
-If you are using nuxt server routes you can enable scalar simply by enabling openAPI in the nitro
-config in your nuxt.config.ts
+If you are using Nuxt server routes, you can enable scalar simply by enabling openAPI in the nitro
```
Comment (JacobCoffee) [inline README.md:124]:
```suggestion
If you are using Nuxt server routes, you can enable scalar simply by enabling `openAPI` in the nitro
```

File: README.md [hunk 3]
```diff
@@ -172,7 +177,7 @@ You can [pass props to customize the API reference](https://github.com/scalar/sc
 
 The API Reference package is written in Vue, that shouldn’t stop you from using
 it in React though! We have created a client side (untested on SSR/SSG) wrapper
```
Comment (JacobCoffee) [inline README.md:179]: consider moving this out of and into some kind of warning admonition:

<details>
<summary>Details</summary>

image

</details>

File: README.md [hunk 4]
```diff
@@ -181,21 +186,24 @@ import React from 'react'
...
 }
 
 export default App
 ```
 
-### With Nextjs
+### With Next.js
 
-Our Next.js handler makes it easy to render a reference, just add it to an Api
+Our Next.js handler makes it easy to render a reference; just add it to an Api
```
Comment (JacobCoffee) [inline README.md:206]:
```suggestion
Our Next.js handler makes it easy to render a reference; just add it to an API
```

File: README.md [hunk 5]
```diff
@@ -586,25 +621,31 @@ Or get more advanced by styling our sidebar!
...
 
-For a before and after example of an updated theme see [`legacyTheme.css`](https://github.com/scalar/scalar/tree/main/packages/themes/src/fixtures/legacyTheme.css) and [`updatedTheme.css`](https://github.com/scalar/scalar/tree/main/packages/themes/src/fixtures/updatedTheme.css) in the [`@scalar/themes`](https://github.com/scalar/scalar/tree/main/packages/themes/) package.
+For a before and after example of an updated theme
+see [`legacyTheme.css`](https://github.com/scalar/scalar/tree/main/packages/themes/src/fixtures/legacyTheme.css)
+and [`updatedTheme.css`](https://github.com/scalar/scalar/tree/main/packages/themes/src/fixtures/updatedTheme.css)
+in the [`@scalar/themes`](https://github.com/scalar/scalar/tree/main/packages/themes/) package.
 
 ## Community
 
 We are API nerds. You too? Let’s chat on Discord: <https://discord.gg/8HeZcRGPFS>
```
Comment (JacobCoffee) [inline README.md:641]: `discord.gg/scalar`?

Comment (JacobCoffee) [inline README.md:641]:
```suggestion
We are API nerds. You too? Let’s chat on Discord: <http://discord.gg/scalar>
```

**Review by marclave — APPROVED** [review]
wow, what a great PR. thanks so much @JacobCoffee 

cant believe we didn't have Litestar listed 😭

**Comment by marclave** [comment]
ah seems like we have prettier running on markdown files 🤔

**Comment by JacobCoffee** [comment]
@marclave does this mean i need to run it or it ran it by accident? I think maybe my IDE did this on save or something

File: README.md [hunk 6]
```diff
@@ -181,21 +187,24 @@ import React from 'react'
...
   return (
     <ApiReferenceReact
-      configuration={{
-        spec: {
-          url: 'https://petstore3.swagger.io/api/v3/openapi.json',
-        },
-      }}
-    />
-  )
+      configuration = {
```
Comment (hanspagel) [inline README.md:190]: I think I’d prefer this to be intended. Wdyt?

**Comment by hanspagel** [comment]
OMG what a great PR, feels like a new README

File: README.md (hunk 6 above)
Comment (JacobCoffee) [reply README.md]: I reverted it :D

**Comment by JacobCoffee** [comment]
Not 100% sure how to run prettier here, I tried manually but gave a weird diff, and also 
```
✗ pnpm prettier README.md
```
which gave
```
ERR_PNPM_OTHER_PM_EXPECTED  This project is configured to use ^pnpm
```
I'm the least frontend savvy person in the world so open to all feedback 😅

**Pull request merged**